### PR TITLE
Missing translations for Catalan (ca)

### LINKED ISSUE #38731: Missing translations for Catalan (ca)
Hello,

There are some missing translation for Catalan (ca). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.ca.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Form/Resources/translations/validators.ca.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.ca.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.ca.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.ca.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.ca.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by rogerguasch**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #38731
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.ca.xlf [hunk 1]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please select a valid timezone.</source>
+                <target>Per favor selecciona una zona horària vàlida.</target>
+            </trans-unit>
+            <trans-unit id="121">
+                <source>Please enter a valid URL.</source>
+                <target>Per favor introdueix una URL vàlida.</target>
+            </trans-unit>
+            <trans-unit id="122">
+                <source>Please enter a valid search term.</source>
+                <target>Per favor introdueix un concepte de cerca vàlid.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.ca.xlf:111]: This (google) translates to "search concept" is that correct? Is "Introduïu un terme de cerca vàlid." better?

(I dont speak Catalan, so feel free to say no)

**Review by Nyholm — COMMENTED** [review]
Awesome. I just have one question.

File: src/Symfony/Component/Form/Resources/translations/validators.ca.xlf (hunk 1 above)
Comment (rogerguasch) [reply src/Symfony/Component/Form/Resources/translations/validators.ca.xlf:111]: For me, both options are correct but I choose "concepte" over "terme".

And with "introduïu" you lose the "please" word.

Per favor = Please
Introdueix / Introduïu -> the second is more polite but the same meaning.
Concepte / terme -> Same meaning again but I think the first is better


Roger

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.ca.xlf:111]: Okey. Thank you for confirming.

**Review by Nyholm — APPROVED** [review]
Thank you

**Comment by derrabus** [comment]
Thank you @rogerguasch.

**Pull request merged**